### DISCUSSION TITLE
Set SM2135 current using command

### DISCUSSION THREAD

**Discussion by VaThInK** [arendst/Tasmota#10570] [Ideas]
I have 3 models of RGBCCT light bulbs (5W, 13W and 15W) that use SM2135 LED driver. 

Found out that I had to modify the current setting for these light bulbs in order to output the correct brigtness.

It would be good if the current setting for component SM2135 can be altered using command instead of compiling custom firmwares for each type, with the lowest current as the default.

**Comment by vic42** [comment]
Probably rather a SetOption than a command.
What values did you find to be correct for the various bulbs?

**Reply by VaThInK** [reply]
Yup, a SetOption would work very well too.

For the 15W bulbs, it needs 45mA and 60mA for RGB and CCT respectively.

I haven't spent the time to find out the correct current for the 5W & 13W bulbs just yet.

I can help with testing the new SetOption if needed. It appears that this chip is not that common in other parts of the world or at least not yet.

**Comment by arendst** [comment]
Fixed. See #10634

**Reply by VaThInK** [reply]
> Fixed. See #10634

Thank you very much for the prompt implementation.

Took the latest build for a spin and it works beautifully on my 15W bulb. When I get the chance I'll do further testing with my other bulbs in the coming days.

Much appreciated!
